### DISCUSSION TITLE
Forbidden 403 error trying to access phpmyadmin in www

### DISCUSSION THREAD

**Discussion by Jodes81** [leokhoa/laragon#469] [Q&A]
Hi,

I'm new here, sorry if this is a dumb question.

I'm using windows 10. I copied phpmyadmin files into a phpmyadmin folder under the www directory. I can access other things in the www directory fine using a browser, but when I try to access phpmyadmin I get "Forbidden You don't have permission to access this resource.". 

I have tried navigating to www, selecting properties on phpmyadmin, going to the security tab, and enabling "full access" on all users, and I also added user "Everyone" and gave them full access. But I still get the error?

I've looked through httpd.conf and sites-enabled and it all seems to look ok. For example, in 'sites-enabled/auto.phpmyadmin.test.conf':
```
define ROOT "C:/laragon/www/phpmyadmin"
define SITE "phpmyadmin.test"
<VirtualHost *:80> 
    DocumentRoot "${ROOT}"
    ServerName ${SITE}
    ServerAlias *.${SITE}
    <Directory "${ROOT}">
        AllowOverride All
        Require all granted
    </Directory>
</VirtualHost>
```
Please help!

**Comment by Pen-y-Fan** [comment]
I don't use phpMyAdmin, I prefer HeidiSQL.

There is already an alias for phpMyAdmin in C:\laragon\etc\apache2\alias\phpmyadmin.conf

The alias is for a none existent directory: C:/laragon/etc/apps/phpMyAdmin/

It may be worth looking at moving phpMyAdmin to this directory.

**Comment by zeeid** [comment]
i'm facing the same issue, any solution  here ?

**Comment by Abdulfortech** [comment]
I also have the same issue, any help please?

**Comment by ghost** [comment]
phpMyAdmin must be under C:\laragon\etc\apps (if you use the default laragon installation directory) and name the folder as phpMyAdmin. Then restarart laragon.

The easy way ist to Replace in TOOLS -> Quick edit ->  the line *phpmyadmin= by the (more generic) link to phpMyAdmin latest version : phpmyadmin=https://www.phpmyadmin.net/downloads/phpMyAdmin-latest-all-languages.zip. Then save the file and exit the editor. Than save and exit the file.
Under  Tools -> Quick add klick phpMyAdmin and after installation you can use it bei cklick at databae
